### ISSUE TITLE
Why images * 2.0 - 1.0 In Plucker Rays calculation?

### ISSUE THREAD

**Issue by chenkangjie1123** [Haian-Jin/LVSM#30]
https://github.com/Haian-Jin/LVSM/blob/ef1dff35484e7be6e66ad6c86089d1b3e9470af8/model/LVSM_scene_decoder_only.py#L189

why not return torch.cat([images, pose_cond], dim=2) ?
